Can't compile in Docker after PR #2845

### Expected behavior
After PR #2845 and PR #2844 (probably only one of them being toxic) I am not able to compile `dev` version with Docker (Windows 10).

### Actual behavior
```
CC app/user/user_main.c
In file included from user_main.c:25:0:
../../sdk-overrides/include/espconn.h:1:1: error: expected identifier or '(' before '.' token
 ../../app/include/lwip/app/espconn.h
 ^
user_main.c: In function 'user_process_input':
user_main.c:298:32: error: 'input_sig' undeclared (first use in this function)
     return task_post_low(input_sig, force);
                                ^
user_main.c:298:32: note: each undeclared identifier is reported only once for each function it appears in
user_main.c: In function 'user_init':
user_main.c:340:5: error: 'input_sig' undeclared (first use in this function)
     input_sig = task_get_id(handle_input);
     ^
make[2]: *** [.output/eagle/debug/obj/user_main.o] Error 1
make[2]: Leaving directory `/opt/nodemcu-firmware/app/user'
make[1]: *** [.subdirs] Error 2
make[1]: Leaving directory `/opt/nodemcu-firmware/app'
make: *** [.subdirs] Error 2
make: Leaving directory `/opt/nodemcu-firmware'
```

### Test code
No test code is needed. I just can't compile clean pull of `dev` version with the error above.

### NodeMCU version
`dev`

### Hardware
N/A

--- a/sdk-overrides/include/espconn.h
+++ b/sdk-overrides/include/espconn.h
@@ -1,1 +1,5 @@
-../../app/include/lwip/app/espconn.h
+#ifndef _SDK_OVERRIDE_ESPCONN_H_
+#define _SDK_OVERRIDE_ESPCONN_H_
+// Pull in the correct lwIP header
+#include "../../app/include/lwip/app/espconn.h"
+#endif

